.github/pull_request_template.md

```diff
@@ -31,8 +31,9 @@ https://raw.githubusercontent.com/ceph/ceph/master/SubmittingPatches.rst
 
 -->
 ## Checklist
-- [ ] References tracker ticket
-- [ ] Updates documentation if necessary
+- [ ] References [tracker ticket](https://tracker.ceph.com/issues/new)
+  - [ ] References tracker ticket to [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new) and [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new) (if necessary)
```

Comment:
actually I have much fewer issues with this. I'd be happy to leave orchestrator out here.

Reply:
@sebastian-philipp noted. Nevertheless, as Dashboard often relies on the underlying Orchestrator support, do you mind if I leave the Orch reference here?